automod_legacy/assets/automod_legacy.html

```diff
@@ -186,31 +186,13 @@ <h3>Basic automod</h3>
 {{define "automod_legacy_banned_websites"}}
 <div class="col-lg-12">
     {{mTemplate "automod_legacy_common_fields" "Guild" .ActiveGuild "Rule" .AutomodConfig.Sites "Name" "Sites"}}
-
     {{checkbox "Sites.GoogleSafeBrowsingEnabled" "automod-google-safebrowsing" `Enable <b>Google safebrowsing</b> integration` .AutomodConfig.Sites.GoogleSafeBrowsingEnabled}}
     <p>Detects sites that contains malware, scams, and other dangerous sites</p>
     <hr />
-    <!-- <p><b>Google safe browsing integration <i>(Coming soon!)</i></b></p> -->
-    <!-- <div class="checkbox">
-            <label>
-                <input type="checkbox" name="Sites.BuiltinBadSites" {{if .AutomodConfig.Sites.BuiltinBadSites }} checked{{end}} disabled> Enable safe browsing malware site detection
-            </label>
-        </div>
-        <div class="checkbox">
-            <label>
-                <input type="checkbox" name="Sites.BuiltinBadSites" {{if .AutomodConfig.Sites.BuiltinBadSites }} checked{{end}} disabled> Enable safe browsing social engineering site detection
-            </label>
-        </div>
-        <div class="checkbox">
-            <label>
-                <input type="checkbox" name="Sites.BuiltinBadSites" {{if .AutomodConfig.Sites.BuiltinBadSites }} checked{{end}} disabled> Enable safe browsing unwanted software site detection
-            </label>
-        </div>
-        <div class="checkbox">
-            <label>
-                <input type="checkbox" name="Sites.BuiltinBadSites" {{if .AutomodConfig.Sites.BuiltinBadSites }} checked{{end}} disabled> Enable safe browsing potentially harmfull application site detection
-            </label>
-        </div> -->
```

Comment:
Can I suggest that this commented code be simply removed if it is no longer needed? (Keeping it there confuses future readers of the code -- is it not working, not useful anymore, or meant to be removed completely?) Those that found it useful can always search back in Git history.

Reply:
This is already removed in this PR! :D